kart/point_cloud/v1.py

```diff
@@ -40,3 +47,52 @@ def tilenames_with_lfs_paths(self):
         """Returns a generator that yields every tilename along with the path where the tile content is stored locally."""
         for blob_name, lfs_hash in self.tilenames_with_lfs_hashes():
             yield blob_name, get_local_path_from_lfs_hash(self.repo, lfs_hash)
+
+    def decode_path(self, path):
+        rel_path = self.ensure_rel_path(path)
+        if rel_path.startswith("tiles/"):
```

Comment:
Maybe(?) it's too late to change this, but since the feature tree is called `feature/` singular it feels like maybe this should be called `tile/` singular too. 🤷